example/integrations/tensorflow/Dockerfile.gpu

```diff
@@ -0,0 +1,12 @@
+# NOTE: the build process would change during developing.
+
+FROM nvidia/cuda:10.0-runtime-ubuntu16.04
+MAINTAINER volcano <maintainer@volcano.sh>
+RUN  apt-get update --fix-missing \
+&& apt-get install -y git python2.7 python-pip libcudnn7=7.4.1.5-1+cuda10.0 libcudnn7-dev=7.4.1.5-1+cuda10.0 \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/*
+RUN pip install tf-nightly-gpu \
```

Comment:
`tf-nightly-gpu` in pip and master branch of tensorflow/benchmarks may change rapidly.

I think we can use a special version/branch of tensorflow and benchmark to make the image more reliable and more reproducible.

Reply:
personally I prefer to use the package without specifying version unless the latest version has some issues/bugs. And for the master issue, I think we need to add a timestamp to the image tag.

Reply:
I think so, we should support at lease one of the timestamp or version/branch.